### PR TITLE
Fe news display articles

### PR DESCRIPTION
**Pull request by yujia1109**

### CODE REVIEW

File: src/components/ArticlesList.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                return (
+                        <li key={article.article_id}>
+                            <Link >
+                            <h3>{article.title}</h3>
+                            </Link>
+                            <p>Author: {article.author}</p>
+                            <p>Topic: {article.topic}</p>
+                            <Votes votes={article.votes}/>
+                            <p>Comments: {article.comment_count}</p>
+                        </li>
```
Comment (pkaodev) [inline src/components/ArticlesList.jsx]: Extracting this into a separate single article component could be helpful down the line

File: src/components/ArticlesList.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import Votes from "./Votes";
+
+const ArticlesList = () => {
+    const [isLoading, setIsLoading] = useState(true);
+    const [articles, setArticles] = useState([])
+    
+    useEffect(() => {
+        getAllArticles().then((data) => {
+            setArticles(data)
+            setIsLoading(false);
```
Comment (pkaodev) [inline src/components/ArticlesList.jsx:13]: good that you are using a loading state.  we would want to to setIsLoading(true) on line 10.5 as well

File: src/components/Votes.jsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const Votes = ({votes}) => {
+    return (
+        <p> Votes: {votes}</p>
+    )
+}
+
+export default Votes;
```
Comment (pkaodev) [inline src/components/Votes.jsx:7]: Any particular reason this was put into it's own component instead of being with the rest of the article data?

**Comment by pkaodev** [comment]
Looks good overall, just a couple of small changes to be done :)

File: src/components/Votes.jsx (hunk 3 above)
Comment (yujia1109) [reply src/components/Votes.jsx:7]: My plan is to handle add votes later, so I created this component.

File: src/components/ArticlesList.jsx (hunk 2 above)
Comment (yujia1109) [reply src/components/ArticlesList.jsx:13]: ok, I’ll add it.

File: src/components/ArticlesList.jsx (hunk 1 above)
Comment (yujia1109) [reply src/components/ArticlesList.jsx]: No problem, I‘ll extract this part to a new component.

**Pull request merged**